src/noisy-qvm.lisp

```diff
@@ -269,14 +268,9 @@ POVMs (e.g. a NOISY-QVM or DENSITY-QVM)."
               (perturb-measurement c p00 p01 p10 p11))))))
 
 (defmethod transition ((qvm noisy-qvm) (instr quil:measure))
-  (multiple-value-bind (ret-qvm counter)
-      ;; perform actual measurement
-      (call-next-method qvm instr)
-    (corrupt-measurement-outcome ret-qvm instr)
-    (values
-     ret-qvm
-     counter)))
-
+  (call-next-method qvm instr)
```

Comment:
Does it matter that the return value of this call used to be percolated up as the return value of this function, whereas now you return the (presumably in-place-modified) argument that this function received?

Reply:
no; i might even argue that returning anything at all might be not good